### PR TITLE
Fixes and improvements

### LINKED ISSUE #45: [$5] UI issue: Page extends past edge of screen
Description:

The Taas Admin app page extends past edge of screen where as the micro-frontends top navigation bar is not.It creates a blank space there.

image

### PR DESCRIPTION
**Pull request by MadOPcode**
Fixes #45 #46 #53.
Implements #50.

### CODE REVIEW

File: src/routes/WorkPeriods/components/PeriodItem/styles.module.scss [hunk 1]
```diff
@@ -40,8 +40,22 @@ tr.container {
...
+      &.toggle {
+        padding-top: 12px;
+      }
+
+      &.daysWorked {
+        padding-top: 5px;
+      }
+    }
+  }
+}
```
Comment (maxceem) [inline src/routes/WorkPeriods/components/PeriodItem/styles.module.scss:55]: @MadOPcode can we avoid using global class names and always use CSS Modules unless it's a third-party library?

File: src/routes/WorkPeriods/components/PeriodListHead/styles.module.scss [hunk 2]
```diff
@@ -43,6 +43,12 @@
       }
     }
 
+    :global(.startDate),
+    :global(.endDate) {
+      padding-left: 10px;
+      padding-right: 10px;
+    }
+
```
Comment (maxceem) [inline src/routes/WorkPeriods/components/PeriodListHead/styles.module.scss:51]: @MadOPcode same here, can we avoid `:global`.

**Review by maxceem — CHANGES_REQUESTED** [review]
@MadOPcode all work good.

The only thing can we avoid using `:global` classnames unless it's for third-party libraries or SVG animation?

I've mentioned some places in comments below.

File: src/routes/WorkPeriods/components/PeriodItem/styles.module.scss (hunk 1 above)
Comment (MadOPcode) [reply src/routes/WorkPeriods/components/PeriodItem/styles.module.scss]: Done.

File: src/routes/WorkPeriods/components/PeriodListHead/styles.module.scss (hunk 2 above)
Comment (MadOPcode) [reply src/routes/WorkPeriods/components/PeriodListHead/styles.module.scss]: And done.

**Review by maxceem — COMMENTED** [review]
@MadOPcode thanks, all perfect now.

**Pull request merged**